tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/publicAsset/PublicAsset.scala

```diff
@@ -15,7 +16,7 @@ import org.apache.james.mailbox.model.ContentType
 import scala.util.Try
 
 object PublicAssetIdFactory {
-  def generate(): PublicAssetId = PublicAssetId(UUID.randomUUID())
+  def generate(): PublicAssetId = PublicAssetId(Generators.timeBasedGenerator().generate())
```

Comment:
Do we really need a new lib for that? How were we doing until now for time uuids?

Reply:
We use Cassandra driver util class to generate time uuid. But this is the API layer so we can not leverage Cassandra driver one.

Reply:
At that cost I prefer using UUID than time UUID for this.